packages/remark-npm2yarn/src/plugin.ts

```diff
@@ -97,9 +97,9 @@ export const remarkNpm2Yarn: Plugin<
 
       if (!node.meta || node.meta === newMetadata) return;
 
-      if (!node.value.startsWith('npm')) {
+      if (!node.value.startsWith('npm') || !node.value.startsWith('npx')) {
```

Comment:
this line has a problem

Reply:
Omg, that's supposed to be a `&&` there.

Reply:
no worries this happens with everybody 😅